apps/hash-graph/lib/graph/src/snapshot/codec.rs

```diff
@@ -83,7 +83,6 @@ impl<T: DeserializeOwned> Decoder for JsonLinesDecoder<T> {
                 serde_json::from_str(&line)
```

Comment:
(comment for just above)
Could we perhaps attach a line number by using `enumerate`, or would the line number not be accurate?

Reply:
That's a good suggestion, thank you! I added it in [Attach current line number](https://github.com/hashintel/hash/pull/2412/commits/fe4e27d11c9ec856d1877226ad07b227bca03261)